Gordon360/Controllers/LostAndFoundController.cs

```diff
@@ -20,6 +22,20 @@ public ActionResult<int> CreateMissingItemReport([FromBody] MissingItemReportVie
             return Ok(ID);
         }
 
+        /// <summary>
+        /// Update Missing component
```

Comment:
Maybe update this comment to be more descriptive, something like "update missing item report with the given id"